recipe/meta.yaml

```diff
@@ -13,11 +13,13 @@ build:
   number: 0
   entry_points:
```

Comment:
I think less <37 is the choice I would prefer.  As we otherwise run into issue on building for 3.7, which we still need to support.  By see the used functionality, it seems to work for 3.7